CONTRIBUTING.md

```diff
@@ -68,6 +68,16 @@ Updating an existing devfile stack is relatively straightforward:
     - Minimally, testing with odo v2 (`odo create`, `odo push`, etc) and odo v3 (`odo init`, `odo dev`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well.
 5) Open a pull request against this repository with your changes.
 
+### Registry Stack Review Process
+
+For every PR, containing updates for registry stacks the review process is: 
+
+* All CODEOWNERS related with the stack are requested to provide their review.
```

Comment:
```suggestion
* All CODEOWNERS related with the stack are required to provide their review.
```

WDYT about making this worded as a requirement as we expect code owners of stacks to review changes to their respective stack?